mkmsdk/MKMClient.py

```diff
@@ -0,0 +1,29 @@
+from oauthlib.oauth1 import Client
+
+
+class MKMClient(Client):
+    """
+    A personalized OAuth1 Client used for Widget Applications
+    """
+
```

Comment:
As I've said, it's better to explain "Why" and not "What I'm doing". You've explained this [here][1] but it's better that this consideration is available in the client. Indeed, this ``Client`` knows the reason why it exists and not the component who's using it.

[1]: https://github.com/evonove/mkm-sdk/pull/3/files#diff-3a81a9a0a76eec2439cb85afb9b91881R75